dep_check/custom_collections.py

```diff
@@ -0,0 +1,15 @@
+from typing import Generic, Iterator, Set, TypeVar
+
+Item = TypeVar("Item")
+
+
+class OrderedSet(dict[Item, None], Generic[Item]):
```

Comment:
What is this ? Leftover home-grown implementation ?

Reply:
only a test, I remove it

Reply:
It seems ok